frontend/src/components/pages/rp-connect/onboarding/onboarding-wizard.tsx

```diff
@@ -74,6 +79,17 @@ export const ConnectOnboardingWizard = ({
     [persistedInputConnectionName]
   );
 
+  const { data: usersList, refetch: refetchUsers } = useListUsersQuery(undefined, {
+    staleTime: LONG_LIVED_CACHE_STALE_TIME,
+    refetchOnWindowFocus: false,
+  });
+
+  const { data: topicList, refetch: refetchTopics } = useLegacyListTopicsQuery(create(ListTopicsRequestSchema, {}), {
+    hideInternalTopics: true,
+    staleTime: LONG_LIVED_CACHE_STALE_TIME,
+    refetchOnWindowFocus: false,
+  });
```

Comment:
Similar to another comment, it's best to invalidate the query in my opinion, but this will also work.